libs/zephyr-metro-plugin/TESTING.md

```diff
@@ -0,0 +1,426 @@
+# Testing Guide for Zephyr Metro Plugin
+
+This guide explains how to manually test the `zephyr-metro-plugin` during development.
+
+## Prerequisites
+
+Before testing, ensure you have:
+
+- **Node.js**: 18 or higher
+- **React Native environment**: Xcode (iOS) and/or Android Studio (Android)
+- **pnpm**: For monorepo package management
+- **Zephyr Cloud account**: For deployment testing (optional for basic functionality testing)
+
+## Building the Plugin Locally
+
+### 1. Build the Plugin
+
+From the monorepo root:
+
+```bash
+# Build the plugin and its dependencies
+pnpm nx build zephyr-metro-plugin
+
+# Or build with dependencies explicitly
+pnpm nx build zephyr-metro-plugin --with-deps
+```
+
+### 2. Watch Mode (Development)
+
+For iterative development, rebuild on changes:
+
+```bash
+# Watch the source files and rebuild
+cd libs/zephyr-metro-plugin
+pnpm exec tsc --watch -p tsconfig.lib.json
+```
+
+## Setting Up a Test React Native Project
+
+### Option A: Using create-zephyr-apps (Recommended)
+
+Generate a new React Native project with Zephyr integration:
+
+```bash
+npx create-zephyr-apps@latest
+# Select "React Native" when prompted
+```
+
+### Option B: Manual Setup
+
+1. **Create a new React Native project:**
+
+   ```bash
+   npx react-native init ZephyrMetroTest
```

Comment:
This is deprecated

- Switch to npx @react-native-community/cli init for the identical behavior.